examples/fluids/index.md

```diff
@@ -618,3 +618,28 @@ numerous terms in the STG formulation.
 | $\{\kappa^n\}_{n=1}^N$ | k^n  | No           | Yes      |
 | $h_i$           | h_i    | Yes          | No   |
 | $d_w$           | d_w    | Yes          | No   |
+
+### Meshing
+
+The flat plate boundary layer example has custom meshing features to better
+resolve the flow. One of those is tilting the top of the domain, allowing for
+it to be a outflow boundary condition. The angle of this tilt is controled by
+`-top_angle`
+
+The primary meshing feature is the ability to grade the mesh, providing better
+resolution near the wall. There are two methods to do this; algorithmically, or
+specifying the node locations via a file. Algorithmically, a base node
+distribution is defined at the inlet (assumed to be $\min(x)$) and then linearly
+stretched/squeezed to match the slanted top boundary condition. Nodes are
+placed such that `-Ndelta` elements are within `-refine-height` of the wall.
+They are placed such that the element height matches a geometric growth ratio
+defined by `-growth`. The remaining elements are then distributed from
+`-refine_height` to the top of the domain linearly in logarithmic space.
+
+Alternatively, a file may be specified containing the locations of each node.
+The file should be newline delimited, with the first line specifying the number
+of points and the rest being the locations of the nodes. The node locations
+used exactly at the inlet (assumed to be $\min(x)$) and linearly
+stretched/squeezed to match the slanted top boundary condition. The file is
+specified via `-y_node_locs_path`. If this flag is given an empty string, then
+the algorithmic approach will be performed.
```

Comment:
Does the length of this file need to be manually kept in sync with the `-dm_plex_box_faces`? If we want it to decouple (seem like a better user interface), then I'd think we should interpolate in this grading file so the mesh covers the domain implied by the grading.

Reply:
Not entirely, no. If the file has more points than the mesh ( so `-dm_plex_box_faces + 1` ), then it will print out a warning, but just use the first `-dm_plex_box_faces + 1` points. If the file has less, then it will error out. 

Doing an interpolation would be interesting, but the primary intention of this is to exactly match an external mesh. If we do the decouple route, I think it should still default to erroring out/printing a warning if the file and `-dm_plex_box_faces` are not synced.

Reply:
Yeah, I'm thinking of switching from linear to quadratic elements without needing to re-prep the inputs. But I'm okay with a clear error message for now.

Reply:
Let me know when this is ready for review and merging.

Reply:
Yeah, it's ready for now if you're ok with the current file reading implementation. I can always switch to the Viewer implementation later, and I think strong STG is a higher priority at the moment.

Reply:
Could you just address the naming of options for this mesh transform component (in the code, yaml, and docs)? Those will end up in input files and it'll be more disruptive to change them later, especially if they aren't grouped now.

Reply:
```suggestion
Alternatively, a file may be specified containing the locations of each node.
The file should be newline delimited, with the first line specifying the number
of points and the rest being the locations of the nodes. The node locations
used exactly at the inlet (assumed to be $\min(x)$) and linearly
stretched/squeezed to match the slanted top boundary condition. The file is
specified via `-platemesh_y_node_locs_path`. If this flag is given an empty string, then
the algorithmic approach will be performed.
```